packages/runtime/container-runtime/src/deltaScheduler.ts

```diff
@@ -117,6 +132,8 @@ export class DeltaScheduler {
                 numberOfOps: this.schedulingLog.numberOfOps,
                 numberOfTurns: this.schedulingLog.numberOfTurns,
                 processingTime: this.schedulingLog.totalProcessingTime,
+                numberOfOpsBefore: this.queueLengthBeforeBatchBegin,
+                numberOfOpsRemaining:  this.deltaManager.inbound.length,
```

Comment:
"idle" event is fired when queue is empty, so numberOfOpsRemaining is not interesting here as it should be always zero.

The most useful metric here is number of ops we processed in total - from very first beginBatch message till getting here. We would need to collect that data manually (somehow), as looking (alone) at inbound queue size at various times is not helpful - ops could be added to inbound queue as processing happens.